srcDoc/tutorials/performance_validation.rst

```diff
@@ -0,0 +1,642 @@
+**Perftest** for **Connext DDS** support the use of the `-scan` parameter. Also notice that we are using
+`-batchSize 0`. We will also later using batching. Lastly, we also removed the `-peer` parameter, the reason
+being that **Connext DDS** uses by default multicast for the discovery phase, so there is no need to specify
+where the counterpart application is.
+
+Since we are using **RTI Connext DDS**, **Perftest** will be choose some *QoS* settings. The best way
+to understand what is being used, is by looking at the initial summary **Perftest** shows:
+
+    .. code::
+
+        RTI Perftest 3.0.0 06ff338 (RTI Connext DDS 6.0.0)
+
+        Mode: THROUGHPUT TEST
+            (Use "-latencyTest" for Latency Mode)
+
+        Perftest Configuration:
+            Reliability: Reliable
+            Keyed: No
+            Publisher ID: 0
+            Latency count: 1 latency sample every 10000 samples
+            Data Size: 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 63000
+            (Set the data size on the subscriber to the maximum data size to achieve best performance)
+            Batching: No (Use "-batchSize" to setup batching)
+            Publication Rate: Unlimited (Not set)
+            Execution time: 20 seconds
+            Receive using: Listeners
+            Domain: 1
+            Dynamic Data: No
+            FlatData: No
+            Zero Copy: No
+            Asynchronous Publishing: No
+            XML File: perftest_qos_profiles.xml
+
+        Transport Configuration:
+            Kind: UDPv4
+            Nic: eth0
+            Use Multicast: False
+
+See below the output results of executing this test. Again, the information displayed here is
+only what the subscriber side showed.
+
+Throughput Results -- RTI Connext DDS Professional (UDPv4) -- No batching
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+    .. csv-table::
+        :align: center
+        :header-rows: 1
+
+        "Size", "Packets", "Packets/s (ave)", "Mbps (ave)", "Lost", "Lost (%)"
+        32,140000,7100,1.8,0,0.00
+        64,140000,6719,3.4,0,0.00
+        128,140000,6680,6.8,0,0.00
+        256,140000,6632,13.6,0,0.00
+        512,110000,5663,23.2,0,0.00
+        1024,110000,5383,44.1,0,0.00
+        2048,100000,4810,78.8,0,0.00
+        4096,60000,2690,88.2,0,0.00
+        8192,30000,1445,94.7,0,0.00
+        16384,20000,720,94.4,0,0.00
+        32768,10000,364,95.6,0,0.00
+        63000,10000,190,96.0,0,0.00
+
+We will discuss the results later, but in **RTI Connext DDS Professional** we have a very
+interesting feature worth mentioning:
+`batching <https://community.rti.com/static/documentation/connext-dds/6.0.0/doc/manuals/connext_dds/html_files/RTI_ConnextDDS_CoreLibraries_UsersManual/index.htm#UsersManual/BATCH_Qos.htm#sending_2410472787_2558262>`__.
+By using this feature we will be able to send more efficiently and improve our
```

Comment:
By using this feature we will be able to send more efficiently by sending several data samples as part of the same packet ...